internal/backends/postgresql/query_iterator.go

```diff
@@ -47,11 +55,16 @@ type queryIterator struct {
 //
 // Nil rows are possible and return already done iterator.
 // It still should be Close'd.
-func newQueryIterator(ctx context.Context, rows pgx.Rows) types.DocumentsIterator {
+func newQueryIterator(ctx context.Context, params *iteratorParams) types.DocumentsIterator {
```

Comment:
Let's not bring previous implementation (hack that I did) here. It complicates things. The implementation specific to `explain` can be not exposed here.

Reply:
Sorry my explanation wasn't clear :disappointed:

There is only one row, we don't need iterator. See this diff if it makes sense :pray: 
 https://github.com/FerretDB/FerretDB/pull/3465#discussion_r1343323383

Reply:
Makes sense, thanks!